zerver/openapi/zulip.yaml

```diff
@@ -11954,6 +11956,15 @@ paths:
                   items:
                     type: integer
                   example: [1, 2]
+                notify_referrer_on_join:
+                  description: |
+                    A boolean indicating whether the referrer has chosen to receive direct notifications
+                    from the notification bot. If `false` the referrer will not receive a notification for
+                    accepted invitations.
+
+                    **Changes**: New in Zulip 9.0 (feature level 254).
```

Comment:
"A boolean indicating whether the referrer would like to receive a direct message from Notification Bot a user account is created using this invitation."

You should also add `default: true` so that information is displayed, and `example: false` seems more appropriate given that default.

Please post screenshots of the resulting API documentation once you've fixed this up.

This changes entry should include "Previously, referrers always received such direct messages."